aksel.nav.no/website/components/assets/Icons.tsx

```diff
@@ -216,3 +216,99 @@ export const CodeSandboxLogo = ({ title, ...props }) => (
     ></path>
   </svg>
 );
+
+export const ApplicationPictogram = ({ title, ...props }) => (
+  <svg
+    xmlns="http://www.w3.org/2000/svg"
+    width="72"
+    height="72"
+    viewBox="0 0 72 72"
+    fill="none"
```

Comment:
I en "oppgradert" variant av primitives vil det være mulig å sette flex-shrink med props, men er ikke mulig akkurat nå. Så tenker en inline style med flexShrink kan være ok frem til da. (Kan evt legge til en TODO for å huske å bytte når det er klart)
```suggestion
    fill="none"
    style={{ flexShrink: 0 }}
```

Reply:
Har endret det, men om ikke ikonet inlines, så blir vel kanskje dette mer "usynlig" for brukerne av malen? Har fremdeles litt lyst til å trekke ut headeren i en egen komponent eksponert gjennom ds-react for å gjøre det enklere å bruke malen. Synspunkter?